docs/source/integrations/sagemaker.rst

```diff
@@ -0,0 +1,11 @@
+=============
+AWS SageMaker
+=============
+
+BentoML makes deploying models to SageMaker easier! Try deploying your model with
+BentoML and bentoctl to SageMaker: https://docs.bentoml.org/en/latest/deployment/aws_sagemaker.html
```

Comment:
This link is broken I believe.

Reply:
this page is hidden rn, will need a complete rewrite